apps/Makefile

```diff
@@ -1,16 +1,13 @@
-TARGETS = bin/amd64-linux/ipfs-driver \
-		bin/amd64-darwin/ipfs-driver
+SUBDIRS	=	ipfs-driver community
```

Comment:
we should talk about how we're eventually going to change this to make it dynamic. is this how we'll build generic apps to or just the ones habitat always runs?

Reply:
This just lets us know which things to build in this repo. Won't be for anything generic. I can write up some documentation on what is expected for a Habitat app to provide. The rest is incidental to this repos build system